modules/sievefilters/modules.php

```diff
@@ -462,7 +462,7 @@ public function process() {
                 Hm_Msgs::add('ERRIMAP Mailbox select error');
                 return;
             }
-            $msg_header = $imap->get_message_headers($form['imap_msg_uid']);
+            $msg_header = $imap->get_message_headers($this->request->post['imap_msg_uid']);
```

Comment:
Can you please switch the usage of request->post variables directly in this whole method to $form array as these should have already been filtered appropriately before using in code. This will be in sync with all the other places Cypht is using process_form to process incoming variables.

Reply:
Notes. Thanks